src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsEnums.cs

```diff
@@ -149,7 +149,7 @@ internal static class TdsEnums
         public const byte SQLDONEINPROC = 0xff;
         public const byte SQLOFFSET = 0x78;
         public const byte SQLORDER = 0xa9;
-        public const byte SQLDEBUG_CMD = 0x60;
+        public const byte SQLDEBUG_CMD = 0x60; // This needs more info
```

Comment:
Who needs to provide the info? the SQL Server team? The spec?

Reply:
I have to provide more info and add it here 😄 . This is just a replacement for more info and triggering pipelines.